lib/slimmer/skin.rb

```diff
@@ -105,6 +105,11 @@ def process(processors,body,template, rack_env)
 
     def success(source_request, response, body)
       wrapper_id = options[:wrapper_id] || 'wrapper'
+
+      # The original request might not be UTF-8, Unicorn will set it to be
```

Comment:
I think this comment could read a little better, seems to be like the 'not' shouldn't be there.